md4c/md4c.h

```diff
@@ -271,6 +279,8 @@ typedef struct MD_SPAN_IMG_DETAIL {
 
 #define MD_FLAG_PERMISSIVEAUTOLINKS         (MD_FLAG_PERMISSIVEEMAILAUTOLINKS | MD_FLAG_PERMISSIVEURLAUTOLINKS | MD_FLAG_PERMISSIVEWWWAUTOLINKS)
 #define MD_FLAG_NOHTML                      (MD_FLAG_NOHTMLBLOCKS | MD_FLAG_NOHTMLSPANS)
+#define MD_FLAG_REDDITSLASHDETECTION             0x8000  /* Enable Reddit autolinks */
+#define MD_FLAG_REDDIT_SLASHES_AS_LINKS        0x4000 //Instead of making Reddit links into special spans, make them into web links
```

Comment:
I think that the logic related to the `MD_FLAG_REDDIT_SLASHES_AS_LINKS` has nothing to do with the **parsing** but only with output **rendering**, and therefore it would be IMHO better to place the related logic just into the md2html utility. Take a look how it treats e.g. `MD_RENDER_FLAG_VERBATIM_ENTITIES`.

Or is there something what I have overlooked?